### PR TITLE
Don't watch forks marked as disabled

### PR DESCRIPTION
**Pull request by clemente**
Small fix to honour the `WatchedFork.enabled` attribute.

### CODE REVIEW

File: pr_watch/tasks.py [hunk 1]
```diff
@@ -49,7 +49,7 @@ def watch_pr():
     organization on the watched repository
     """
     try:
-        for watched_fork in WatchedFork.objects.all():
+        for watched_fork in WatchedFork.objects.filter(enabled=True):
```
Comment (pomegranited) [inline pr_watch/tasks.py]: Thanks for spotting this @clemente !

Just needs a test added to https://github.com/open-craft/opencraft/blob/master/pr_watch/tests/test_tasks.py

**Comment by pomegranited** [comment]
@clemente :+1: 

- [x] I tested this by using the test instructions from https://github.com/open-craft/opencraft/pull/246, and verifying that an de-activating the `WatchedFork` causes the `WatchedPullRequest` to not be created.
  Also ensured that the unit tests run without error.
- [x] I read through the code
- [x] <strike>I checked for accessibility issues</strike>
- [x] Includes documentation - N/A (field is straightforward)

**Comment by UmanShahzad** [comment]
@clemente Can you squash/rebase? We can stop the current build.

**Comment by clemente** [comment]
Squashed and rebased.

**Pull request merged**